blackbox/docs/administration/hba.txt

```diff
@@ -0,0 +1,78 @@
+.. _administration_hba:
+
+=========================
+Host Based Authentication
+=========================
+
+.. NOTE::
+
+   Host based Authentication is an Enterprise Edition feature.
+   This feature only concerns connections over ``PostgreSQL Wire Protocol``.
+
+This topic explains how to configure client connections and authentication for CrateDB.
+
+When CrateDB is started, the cluster contains one predefined ``superuser``. This user
+is referred to as ``crate``.
+
+By default, CrateDB’s configuration does not contain any host based authentication
+entries. The authentication is therefore disabled, and all users are allowed to connect.
+
+To allow CrateDB access from specific hosts, you need to configure the
+``auth.host_based`` setting in the ``crate.yml`` file.
+
+Allowing Access to CrateDB
+--------------------------
+
+Client access and authentication over the ``PostgreSQL Wire Protocol`` is administered
+by ``auth.host_based`` configuration in the ``crate.yml`` file.
+
+The general format of the ``auth.host_based`` setting is a list of remote client access
+entries, one per line, that users can specify to restrict access to CrateDB. Each entry
+is a is a dictionary that contains a user, an address and a method. The description of
+these fields can be found in :ref:`host_based_auth:`.
+
+For example, a host based configuration can look like this::
+
+    auth:
+      Host_based:
+        - {user: mike, address: 32.0.0.0/8, method: trust}
+        - {user: barb, address: 172.16.0.0}
+        - {user: eleven}
+        - {method: trust}
+
+.. NOTE::
+
+   In the ``auth.host_based`` setting, the order of entries is relevant.
+   The first entry that matches the client user and address will be used.
```

Comment:
The authentication method of the first entry that matches the client user and address will be used. If the authentication is not successful, subsequent entries will not be considered.